### ISSUE TITLE
Product donation page comp

### ISSUE THREAD

**Issue by schuyler1d** [MoveOnOrg/giraffe#133]
https://github.com/MoveOnOrg/ak-template-set/issues/476
has a page example here:
https://act.moveon.org/donate/our-fight_copy/

MoveOn mostly uses store.moveon.org but occasionally this is used (apparently), so would be good to comp something while we're in deep on donations.

Marking as 'ActionKit components' milestone since it's lower priority than the other donation pieces.

**Comment by hi0ctane** [comment]
hey @schuyler1d attached is a comp for the table scenario!

![donation-form-table](https://user-images.githubusercontent.com/5332564/38096948-216e8b08-3342-11e8-8306-6db2280a0f88.jpg)

**Comment by schuyler1d** [comment]
awesome!  closing and unblocked the ak-template-set issue
https://github.com/MoveOnOrg/ak-template-set/issues/476
